modules/sso.php

```diff
@@ -651,11 +704,23 @@ function handle_login() {
 				setcookie( 'jetpack_sso_redirect_to', ' ', time() - YEAR_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN );
 			}
 
-			wp_safe_redirect( apply_filters( 'login_redirect', $redirect_to, $_request_redirect_to, $user ) );
+			wp_safe_redirect(
+				/**
```

Comment:
This should be "This filter is documented in core/src/wp-login.php" ( https://github.com/WordPress/WordPress/blob/master/wp-login.php#L820-L828 )